### ISSUE TITLE
False positive on xstate/spawn-usage

### ISSUE THREAD

**Issue by sbaechler** [rlaffers/eslint-plugin-xstate#16]
**Describe the bug**
It looks like the rule xstate/spawn-usage does not check if the `spawn` function is actually imported from xstate.

It reports an error in all my redux sagas because redux-saga also has an effect called `spawn`.

**Expected behavior**
The rule checks if the `spawn` function is the one from xstate.

**Actual behavior**
Any function called `spawn` is considered.

**Versions (please complete the following information):**
 - Node version: 18.12.1
 - ESLint version: 8.44.0
 - eslint-plugin-xstate version: 1.1.0

**Additional context**
I am getting a lot of false positives in one of my saga files.
The import line is:

    import { all, call, delay, fork, getContext, put, race, select, spawn, take, takeEvery } from 'redux-saga/effects';

There is no import from xstate in this file.

**Comment by rlaffers** [comment]
Thanks for the report. I'm going to look into how the source of an imported symbol can be checked in Eslint. In the meantime, you can opt-out from this rule with:
```
// eslint-disable-next-line xstate/spawn-usage
```

**Comment by rlaffers** [comment]
This was fixed in https://github.com/rlaffers/eslint-plugin-xstate/releases/tag/v1.1.1
